examples/04_pong/systems/winner.rs

```diff
@@ -12,14 +12,12 @@ use audio::Sounds;
 pub struct WinnerSystem;
 
 impl<'s> System<'s> for WinnerSystem {
-    type SystemData = (
-        WriteStorage<'s, Ball>,
-        WriteStorage<'s, LocalTransform>,
-        FetchMut<'s, ScoreBoard>,
-        Fetch<'s, AssetStorage<Source>>,
-        Fetch<'s, Sounds>,
-        Fetch<'s, Option<Output>>,
-    );
+    type SystemData = (WriteStorage<'s, Ball>,
+     WriteStorage<'s, LocalTransform>,
```

Comment:
I feel like we change this formatting in every PR, and every contributor says they're using the latest version..

Reply:
rustup update
cargo +nightly install rustfmt --force
cargo +nightly fmt

Reply:
should be `cargo +nightly install --force rustfmt-nightly`

Reply:
and `cargo +nightly fmt --all`

Reply:
Fixed